localstack-core/localstack/services/transcribe/provider.py

```diff
@@ -44,6 +44,8 @@
 from localstack.utils.run import run
 from localstack.utils.threads import start_thread
 
+MAX_AUDIO_DURATION_SECONDS = 60 * 60 * 4
```

Comment:
We can add a link in the comments here pointing to the AWS documentation specifying the maximum audio file length here.

Reply:
I have add the link now =)